TOMP-API.yaml

```diff
@@ -2372,7 +2378,7 @@ components:
       schema:
         type: string
       description: Version of the API.
-      example: 0.5.1
+      example: 0.5.2
```

Comment:
Haha, maybe we should should make this a true example number like 1.23.45 so we won't have to update the version in two places each time.